src/css/base.scss

```diff
@@ -1,18 +1,125 @@
 * {
   margin: 0;
+  font-family: 'Noto Sans', sans-serif;
 }
 
-img {
-  height: 100px;
-  width: auto; 
+// @import 'styles';
+@import '~js-datepicker/src/datepicker';
+$darkGray: darkgray;
```

Comment:
An alternative would be to name this variable `dark` or `mainColor` or something like that so that if you change your mind about the color later, the variable name won't be a lie 😄